zinc/src/sbt-test/source-dependencies/trait-trait-211/build.json

```diff
@@ -0,0 +1,8 @@
+{
+  "projects": [
+    {
+      "name": "mirtest",
+      "scalaVersion": "2.11.8"
```

Comment:
Nit: latest version is 2.11.x.

Reply:
This is intentionally downgraded to 2.11.8 to match the Scala version used by Quasar.